Assets/SEE/GameObjects/CityCursor.cs

```diff
@@ -33,29 +50,46 @@ private void Start()
             }
         }
 
+        /// <summary>
+        /// Unsubscribes from events when this object is destroyed
+        /// and destroys the <see cref="Cursor"/>.
+        /// </summary>
         private void OnDestroy()
         {
-            InteractableObject.AnySelectIn -= AnySelectIn;
-            InteractableObject.AnySelectOut -= AnySelectOut;
-
-            Destroyer.Destroy(E);
+            InteractableObject.AnyHoverIn  -= AnyHoverIn;
+            InteractableObject.AnyHoverOut -= AnyHoverOut;
+            Destroyer.Destroy(Cursor);
         }
 
-        private void AnySelectIn(InteractableObject interactableObject, bool isInitiator)
+        /// <summary>
+        /// Makes <paramref name="interactableObject"/> the <see cref="Cursor"/> focus when
+        /// it belongs to <see cref="city"/>.
+        /// Called whenever an <see cref="InteractableObject"/> is selected.
+        /// </summary>
+        /// <param name="interactableObject">the selected object</param>
+        /// <param name="isInitiator">currently ignored</param>
+        private void AnyHoverIn(InteractableObject interactableObject, bool isInitiator)
```

Comment:
If a parameter is completely unused, and we just add it to comply with a specified signature, we can mark it as throw-away by renaming it to `_` (then we can also remove the `<param` line, since the intent is clear from the name alone). Also applies to the method below.

Reply:
Fixed.